tests/tor_exit_nodes_test.py

```diff
@@ -0,0 +1,126 @@
+from unittest.mock import patch, MagicMock
+from core import database_arango
+from core.schemas.observable import ObservableType
+from tests.helpers import TestHelpers
+from core.schemas.observables import ipv4
+from plugins.feeds.public.tor_exit_nodes import TorExitNodes
+
+
+class TorExitNodesTest(TestHelpers):
```

Comment:
If you follow the recommendation above, you should also change this class' base class